modules/ebi-metagenomics/antismash/main.nf

```diff
@@ -0,0 +1,67 @@
+
+process ANTISMASH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // current antiSMASH bioconda container is buggy so we use our custom one (https://github.com/antismash/antismash/discussions/711)
+    container "microbiome-informatics/antismash:7.1.0.1_2"
+
+    input:
+    tuple val(meta), path(contigs)
+    tuple val(meta), path(genes)
+    tuple path(antismash_db), val(db_version)
+
+    output:
+    tuple val(meta), path("${meta.id}_results/${meta.id}.gbk")  , emit: gbk
+    tuple val(meta), path("${meta.id}_results/${meta.id}.json") , emit: json
+    tuple val(meta), path("${meta.id}_antismash.tar.gz")        , emit: results_tarball
+    path "versions.yml"                                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def is_compressed = antismash_db.name.endsWith(".tar.gz")
+    def antismashdb_name = antismash_db.name.replace(".tar.gz", "")
```

Comment:
Should we support a compressed `antismash_db`? For our pipeline it will introduce an significant overhead.

Reply:
makes sense, I will add that